### ISSUE TITLE
"indention"

### ISSUE THREAD

**Issue by nxskok** [r-lib/styler#509]
Isn't the word "indentation" rather than "indention"? For example in the function name `specify_reindention`.

**Comment by jonmcalder** [comment]
You are right that `indentation` is far more common, and I also thought this was a weird choice of word at first, but I think that `indention` may actually be most appropriate here, if nothing else because it is the shortest form of the two possible alternatives.

It seems to be a more archaic version of the term `indentation`, but on the other hand it also seems to refer specifically to the act of indenting text, whereas `indentation` can also be used to refer to other types of indentation (e.g. a recess or sharp depression in any surface).

[AskDifference](https://www.askdifference.com/indention-vs-indentation/)

[WikiDiff](https://wikidiff.com/indention/indentation)

[Google Books - comparison of relative usage](https://books.google.com/ngrams/graph?content=indentation%2Cindention&case_insensitive=on&year_start=1800&year_end=2008&corpus=15&smoothing=7&share=&direct_url=t4%3B%2Cindentation%3B%2Cc0%3B%2Cs0%3B%3Bindentation%3B%2Cc0%3B%3BIndentation%3B%2Cc0%3B.t4%3B%2Cindention%3B%2Cc0%3B%2Cs0%3B%3Bindention%3B%2Cc0%3B%3BIndention%3B%2Cc0%3B%3BINDENTION%3B%2Cc0)

**Comment by lorenzwalthert** [comment]
Thanks @jonmcalder for jumping in, I hoped some native speaker could justify our choice 😄. To be honest, I can't remember it was a deliberate choice. But I think we should stick to it.

**Comment by lorenzwalthert** [comment]
Any more questions @nxskok? Feel free to re-open.

**Comment by nstjhp** [comment]
I may as well ask as this caught me out. Can you make "indentation" a synonym of your "indention"? For example so that `scope = "indentation"` is equivalent to `scope = "indention"`, in the same way Hadley allows `colour` or `color` in ggplot.

**Comment by lorenzwalthert** [comment]
Maybe we should just convert it internally yes.
